IdentityCore/src/network/request/MSIDAADAuthorizationCodeGrantRequest.m

```diff
@@ -42,6 +43,7 @@ - (instancetype)initWithEndpoint:(NSURL *)endpoint
         
         NSMutableDictionary *parameters = [_parameters mutableCopy];
         parameters[MSID_OAUTH2_CLIENT_INFO] = @YES;
+        parameters[MSID_OAUTH2_CLAIMS] = claims ? claims : nil;
```

Comment:
nit: can we simplify it to `parameters[MSID_OAUTH2_CLAIMS] = claims` ?

Reply:
why set if nil?
let's use 
- (void)msidSetObjectIfNotNil:(id)dictObject forKey:(id<NSCopying>)dictKey;

Reply:
I think we can remove `msidSetObjectIfNotNil`.  When we provide 'nil' to the mutable dictionary in subscript, like this:
```
parameters[MSID_OAUTH2_CLAIMS] = nil;
```

It will not crash but remove value under such key. It crashes only when we provide nil to `setObject:forKey`.

Reply:
+1 for what serhii says :)

Reply:
The ternary operator has planted a seed in my mind, I used to make the same comment on Mike's PR, now I did it unconsciously, hope you guys won't be affected 😄

Reply:
fixed